src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/ClientBaseTests.cs

```diff
@@ -125,31 +125,74 @@ public static void ClientMessageInspector_Verify_Invoke()
 
     [Fact]
     [OuterLoop]
-    public static void ClientBaseOfT_Call()
+    public static void ClientBaseOfT_HttpTransport_Call()
     {
```

Comment:
Nit: my bad for choosing a poor name initially.  Would prefer something like ClientBaseOfT_Synchronous_RoundTrip_Call_Using_UsingHttpTransport or something like that